libs/util/data-fetcher/src/lib/readers/gml.ts

```diff
@@ -0,0 +1,50 @@
+import { BaseFileReader } from './base-file'
+import { DataItem, PropertyInfo } from '../model'
+import { fetchDataAsText, processItemProperties } from '../utils'
+import { GeoJSON, WFS } from 'ol/format'
+
+export function parseGml(
+  text: string,
+  namespace: string
+): {
+  items: DataItem[]
+  properties: PropertyInfo[]
+} {
+  const splittedNamespace = namespace.split(':')
+  const regex = new RegExp(`xmlns:${splittedNamespace[0]}=["']([^'"]*)["']`)
+  const match = regex.exec(text)
+
+  if (match && match.length >= 2) {
+    const wf = new WFS({
+      featureNS: match[1],
+      featureType: splittedNamespace[1],
+      version: '2.0.0',
```

Comment:
Actually we don't know what is the version here.
You guess that the default version of the map server is 2.0.0, don't you ?
Is the 2.0.0 the default version of ogc-client, or does ogc-client rely on the version of the getCapabilities ? BTW, do you force a version while call the getCapabilities ?